protocol-units/mock-assets/devnet/m1/mevm/.github/workflows/test.yml

```diff
@@ -0,0 +1,34 @@
+name: test
+
+on: workflow_dispatch
+
+env:
+  FOUNDRY_PROFILE: ci
+
+jobs:
+  check:
+    strategy:
+      fail-fast: true
+
+    name: Foundry project
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+        with:
+          submodules: recursive
+
+      - name: Install Foundry
```

Comment:
Use the nix shell so that we don't have to deal with version drift.

Reply:
do you by any chance have a file reference for this?

Reply:
What do you mean by file reference? Enter the nix shell in the workflow action. Review how this is done for MCR.